PurchasesCoreSwift/Public/Error Handling/ErrorUtils.swift

```diff
@@ -107,6 +107,18 @@ import StoreKit
 
     }
 
+    /**
+     * Constructs an Error with the [ErrorCode.productDiscountMissingIdentifierError] code.
+     *
+     * @note This error code is used when attemping to post data about product discounts but the discount is
+     * missing an indentifier.
+     */
+    @objc public static func productDiscountMissingIdentifierError() -> Error {
+
```

Comment:
do we need the extra lines even for functions?

Reply:
Nope, I was following the style that already exists in this file without even considering it. I'll update the file.